pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDConfiguration.java

```diff
@@ -326,10 +349,18 @@ public Renderer getRenderer() {
         return renderer;
     }
 
+    /**
+     * @deprecated Use {@link #getCPDReportRenderer()} instead
+     */
+    @Deprecated
     public CPDRenderer getCPDRenderer() {
         return cpdRenderer;
     }
 
+    CPDReportRenderer getCPDReportRenderer() {
```

Comment:
This is package-private only, so I'll change the deprecation notices above to internal API. We can make it public API, once there is a use case